packages/api/cms-api/src/dam/files/files.service.ts

```diff
@@ -1,8 +1,10 @@
-import { MikroORM } from "@mikro-orm/core";
+import { MikroORM, Utils } from "@mikro-orm/core";
 import { InjectRepository } from "@mikro-orm/nestjs";
 import { EntityRepository, QueryBuilder } from "@mikro-orm/postgresql";
 import { forwardRef, Inject, Injectable, Logger } from "@nestjs/common";
+import * as console from "console";
```

Comment:
what is that dependency?

Reply:
That's an accidental import that was automatically added by the IDE. I removed it